client/lib/plans/index.js

```diff
@@ -214,11 +203,111 @@ export function getYearlyPlanByMonthly( planSlug ) {
  * @return {Boolean}           Whether the plan "types" match regardless of interval
  */
 export function planLevelsMatch( planSlugA, planSlugB ) {
-	return (
-		planSlugA === planSlugB ||
-		getMonthlyPlanByYearly( planSlugA ) === planSlugB ||
-		planSlugA === getMonthlyPlanByYearly( planSlugB )
-	);
+	const planA = getPlan( planSlugA );
+	const planB = getPlan( planSlugB );
+	return planA && planB && planA.type === planB.type && planA.group === planB.group;
+}
+
+export function isBusinessPlan( planSlug ) {
+	return getPlan( planSlug ).type === TYPE_BUSINESS;
```

Comment:
This type of operation can be safer. If the `planSlug` is invalid, the popular error `TypeError: ‘undefined’ Is Not an Object` will occur.

Reply:
I want to make `getPlan` throw an explicit error if planSlug is invalid - this would mean less time spent on debugging troublesome calls to `getPlan`.